base_tier_validation_helper/models/tier_definition_helper.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError, ValidationError
+
+
+class TierDefinitionHelper(models.Model):
+    _name = 'tier.definition.helper'
+    _description = 'Tier Definition Helper'
+
+    @api.model
+    def _get_default_name(self):
+        return _('New')
+
+    name = fields.Char(
+        string='Description',
+        required=True,
+        default=lambda self: self._get_default_name(),
```

Comment:
```suggestion
        default='New',
```
I just think it will be less code for almost the same thing.

Reply:
Fixed